packages/cli/src/commands/create/index.ts

```diff
@@ -143,7 +144,20 @@ async function _createTutorial(flags: CreateOptions): Promise<undefined> {
 
   await copyTemplate(resolvedDest, flags);
 
-  updatePackageJson(resolvedDest, tutorialName, flags);
+  const provider = await prompts.select({
+    message: 'Select hosting providers for automatic configuration:',
+    options: [
+      { value: 'Vercel', label: 'Vercel' },
+      { value: 'Netlify', label: 'Netlify' },
+      { value: 'Cloudflare', label: 'Cloudflare' },
+      { value: 'skip', label: 'Skip hosting configuration' },
+    ],
+    initialValue: 'Vercel',
+  });
```

Comment:
The test cases are stuck as this prompt is waiting for selection. We need to add CLI flag that can be passed on by pass this, e.g. `--no-provider`.

Similar to `--git` and `--no-git`:

https://github.com/stackblitz/tutorialkit/blob/b2384ffb7ff8ce24f42dafe6d6c47e4fcf0d2384/packages/cli/src/commands/create/git.ts#L12-L18

https://github.com/stackblitz/tutorialkit/blob/b2384ffb7ff8ce24f42dafe6d6c47e4fcf0d2384/packages/cli/src/commands/create/index.ts#L31

Then pass it here in tests:

https://github.com/stackblitz/tutorialkit/blob/b2384ffb7ff8ce24f42dafe6d6c47e4fcf0d2384/packages/cli/tests/create-tutorial.test.ts#L43